tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/util/LsBenchFileUtil.java

```
@@ -0,0 +1,22 @@
+package org.logstash.benchmark.cli.util;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+
+public final class LsBenchFileUtil {
+
+    public static void ensureDeleted(final File file) throws IOException {
+        if (file.exists()) {
+            if (file.isDirectory()) {
```

May want to add ``` file.deleteOnExit();``` and swallow any errors here in favor of a log message.